### DISCUSSION TITLE
how to clear graphics

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#8510] [Q&A]
`        const graphics = new PIXI.Graphics();

        graphics.lineStyle(2, 0xFFFFFF, 1);
        graphics.drawRect(-15, -15,bunny.width+30, bunny.height+30);
        graphics.endFill();
        graphics.name="biankuang";

        selcontainer.addChild(graphics);`
how to clear it

**Comment by bigtimebuddy** [comment]
graphics.clear()
